blueprints/ember-bootstrap/index.js

```diff
@@ -178,13 +180,24 @@ module.exports = {
     let file = 'ember-cli-build.js';
     let bootstrapVersion = this.bootstrapVersion;
     let preprocessor = this.preprocessor;
+    let config = this.retrieveExistingConfiguration();
     let settings = {
       bootstrapVersion
     };
 
-    settings.importBootstrapFont = (bootstrapVersion === 3);
+    if (bootstrapVersion === 4) {
+      settings.importBootstrapFont = false;
+    } else if (config.hasOwnProperty('importBootstrapFont')) {
+      settings.importBootstrapFont = config.importBootstrapFont;
+    } else {
+      settings.importBootstrapFont = true;
+    }
 
-    settings.importBootstrapCSS = (preprocessor === 'none');
+    if (config.hasOwnProperty('importBootstrapCSS')) {
```

Comment:
Do we want to have the same logic here as for `importBootstrapFont`, like first check if we need to override because it is the only possible setting. Here `importBootstrapCSS` is only used if there is no preprocessor, so we could make it like
```js
if (preprocessor !== 'none') {
  settings.importBootstrapCSS = false; 
} else if (config.hasOwnProperty('importBootstrapCSS')) {
  settings.importBootstrapCSS = config. importBootstrapCSS;
} else {
  settings.importBootstrapCSS = true;
}
```